packages/package_info/example/lib/main.dart

```diff
@@ -54,6 +56,22 @@ class _MyHomePageState extends State<MyHomePage> {
       version = 'Failed to get version.';
     }
 
+    String buildNumber;
+    // Platform messages may fail, so we use a try/catch PlatformException.
```

Comment:
They won't fail unless you have code on the Android or iOS side that may fail. The communication mechanism itself is reliable.

Reply:
@mravn-google  The refactor pr is comming: #360 
Please review.